src/frontend/src/common/api/index.tsx

```diff
@@ -29,6 +29,7 @@ export const DEFAULT_FETCH_OPTIONS: RequestInit = {
 export const DEFAULT_PUT_OPTIONS: RequestInit = {
   credentials: "include",
   method: "PUT",
+  ...DEFAULT_HEADERS_MUTATION_OPTIONS,
```

Comment:
Thank you! Minor additional request:
Could you go to `src/frontend/src/common/queries/auth.ts` and remove the matched `DEFAULT_HEADERS_MUTATION_OPTIONS` spread in that file? It was there to accomplish the same thing in a specific way that is now being done in a general way with your change here of including  `DEFAULT_HEADERS_MUTATION_OPTIONS` inside of `DEFAULT_PUT_OPTIONS`. Please and thank you!

Reply:
oh yes good catch will do!